src/Card/README.md

```diff
@@ -405,7 +415,11 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 
   return (
     <Card style={{ width: isExtraSmall ? "100%" : "25rem" }}>
-      <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" srcAlt="Card image"/>
+      <Card.ImageCap
+        skeletonHeight={140} 
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?